### PR TITLE
Update keycloak-authorization and oidc pom descriptions

### LINKED ISSUE #5904: Clean the keycloak authorization extension description
Right know the description on code.quarkus.io is "Build parent to bring in required dependencies"
Which sucks. So let's update it to know what it does and when to include it.

Descriptions are taken from `pom.xml` description or from the `quarkus-extension.yaml`

### PR DESCRIPTION
**Pull request by sberyozkin**
Fixes #5904

### CODE REVIEW

**Comment by sberyozkin** [comment]
Hi @gsmet perhaps the CI build can be disabled for this PR ?

File: extensions/keycloak-authorization/runtime/pom.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
     <artifactId>quarkus-keycloak-authorization</artifactId>
     <name>Quarkus - Keycloak Authorization - Runtime</name>
+    <description>Policy enforcer which uses permissions managed by Keycloak to control access to protected resources</description>
```
Comment (gsmet) [inline extensions/keycloak-authorization/runtime/pom.xml]:
```suggestion
    <description>Policy enforcer using Keycloak-managed permissions to control access to protected resources</description>
```
would be a bit shorter? WDYT?

**Comment by gsmet** [comment]
I squashed the 2 commits.

**Comment by sberyozkin** [comment]
@gsmet I'm not sure why a code flow test started failing now, we've only updated the pom desciptions...

**Review by pedroigor — APPROVED** [review]

**Comment by pedroigor** [comment]
@sberyozkin HtmlUnit may give a false alarm from time to time as we are testing browser flows. I'll keep an eye on it too and if it happens too often (or more than what the team expects) I'm going to review the tests.

**Review by gsmet — APPROVED** [review]

**Pull request merged**